translator/config/os.go

```diff
@@ -22,11 +22,14 @@ func ToValidOs(os string) string {
 		// Give it a last try, using current osType type
 		os = runtime.GOOS
 	}
+
 	formattedOs := strings.ToLower(os)
 	for _, val := range supportedOs {
 		if formattedOs == val {
 			return formattedOs
 		}
 	}
-	panic(fmt.Sprintf("%v is not a supported osType type", os))
+
+	log.Panicf("E! %v is not a supported osType type", os)
+	return ""
```

Comment:
This is unreachable. Frankly, I dislike this because the logic here is fragile. I think that this function should not panic at all. It should either:
1. return an empty string and an error if the input is not a valid OS
2. just return an empty string and code that uses it needs to know to check for the empty string when using the return value from this function.

If that's too much to add to this PR, I would revert the changes to this function and address it separately.

Reply:
Agree on fixing it in a separate PR and fixed it in the newest commit.